backend/test_firestore.go

```diff
@@ -70,15 +76,17 @@ func GetUser(w http.ResponseWriter, r *http.Request) {
 	user := &firebase.User{FirebaseID: vars["id"]}
 	err := user.Get()
 	if err != nil {
-		w.WriteHeader(http.StatusNotFound)
```

Comment:
why did this get removed

Reply:
Using this instead (2 lines below). This was an artifact I found when I was going through.
http.Error(w,"Could not find user with the given id",http.StatusNotFound)